src/firmware.cpp

```diff
@@ -2,51 +2,33 @@
 
 #include <Adafruit_PWMServoDriver.h>
 
-// Multiplying by this converts round-trip duration in microseconds to distance to object in millimetres.
-static const float ULTRASOUND_COEFFICIENT = 1e-6 * 343.0 * 0.5 * 1e3;
-
-static const String FIRMWARE_VERSION = "SourceBots PWM/GPIO v0.0.1";
-
-typedef String CommandError;
+static Adafruit_PWMServoDriver SERVOS = Adafruit_PWMServoDriver();
```

Comment:
Probably worth adding Doxygen docstrings for the functions and a doc block a the top of the file